nodeadm/internal/containerd/default-base-runtime-spec.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ociVersion": "1.1.0",
+  "process": {
+    "rlimits": [
+      {
+        "type": "RLIMIT_NOFILE",
+        "soft": 65536,
+        "hard": 1048576
```

Comment:
appreciate making this configurable, do we have to set a default?  
asking based on the time we updated limits in https://github.com/awslabs/amazon-eks-ami/pull/1535 in al2, but I see the rationale on making this a breaking change to set the default and follow bottlerocket's example

Reply:
Yeah, we have to set a lower default in order to fix #1746.

The issues caused by #1535 were mostly related to the `1024` effective soft limit (which is the "correct" config but is nevertheless incompatible with a lot of software written in the container age). That's why we use a much higher soft limit here, as well as a higher hard limit. I think this is fairly unlikely to be disruptive; and there's an easy mitigation if it does (overriding the limits with something appropriate for the specific workload).